client/environment_import.go

```diff
@@ -0,0 +1,96 @@
+package client
+
+type EnvironmentImport struct {
+	Id             string     `json:"id"`
+	Name           string     `json:"name"`
+	OrganizationId string     `json:"organizationId"`
+	Workspace      string     `json:"workspace"`
+	Variables      []Variable `json:"variables"`
+	GitConfig      GitConfig  `json:"gitConfig"`
+	IacType        string     `json:"iacType"` // opentofu or terraform
+	IacVersion     string     `json:"iacVersion"`
+}
+
+type Variable struct {
+	Name        string `json:"name"`
+	Value       string `json:"value"`
+	IsSensitive bool   `json:"isSensitive"`
+	Type        string `json:"tyoe"` // string or JSON
```

Comment:
tyoe to type

Reply:
[4765a44](https://github.com/env0/terraform-provider-env0/pull/855/commits/4765a4404d264ba91ca48a408173a40cfab9132f)